### PR TITLE
feat(components/forms): hide required asterisk from screen readers

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[#AB2599980](https://blackbaud.visualstudio.com/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2599980)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1982/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/a11y-storybook/)
- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/colorpicker-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/core-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/1982/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1982/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1982/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1982/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1982/playground/)

File: libs/components/theme/src/lib/styles/_public-api/_compat/_mixins.scss [hunk 1]
```diff
@@ -142,6 +142,7 @@
 
 @mixin sky-required-field {
   content: '*';
+  content: '*' / '';
```
Comment (Blackbaud-SandhyaRajasabeson) [inline libs/components/theme/src/lib/styles/_public-api/_compat/_mixins.scss:146]: Asterisk still shows up on all the following browsers. Support for this way of hiding it from screen readers:
Mac
- Safari 
- Chrome ✅
- Firefox
- Edge ✅

Windows
- Chrome ✅
- Firefox 
- Edge ✅

Comment (Blackbaud-SteveBrush) [reply libs/components/theme/src/lib/styles/_public-api/_compat/_mixins.scss:146]:
```
/* comment here */
content: '*';
content: '*' / '';
```

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/1995

**Pull request merged**